cmake/Platform/Linux/Packaging_linux.cmake

```diff
@@ -48,6 +48,7 @@ elseif("$ENV{O3DE_PACKAGE_TYPE}" STREQUAL "DEB")
         libxkbcommon-dev                        # For xcb keyboard input
         libxcb-xfixes0-dev                      # For mouse input
         libxcb-xinput-dev                       # For mouse input
+        libxcb-image0-dev                       # For xcb image support
```

Comment:
If we are adding another xcb package dependency, then it also needs to be added to the build node initialization list, i.e. https://github.com/o3de/o3de/blob/development/scripts/build/build_node/Platform/Linux/package-list.ubuntu-jammy.txt . 
And also in order for AR to run successfully, the linux nodes will need to be rebuilt first with the new nodes, which means that change may need a separate PR to go in first, and then coordination with sig-build is needed to propogate the changes.

Reply:
@Fireronin could you prepare a separate PR as @spham-amzn proposed and link here.

Reply:
Here is pr you asked for

https://github.com/o3de/o3de/pull/17925

Reply:
This will also require an update the documentation [here](https://github.com/o3de/o3de.org/blob/5291e85d708ff2955c43cddd52dc918b6c27d1da/content/docs/welcome-guide/requirements.md?plain=1#L160-L182):

Reply:
Also note that you will need to work with sig-build to get the changes to the AR jenkins nodes from https://github.com/o3de/o3de/pull/17925 before this change, otherwise it will not pass AR, unless you make the splash screen optional (See the additional comment in this review)